services/core-web/src/components/modalContent/StartDraftPermitModal.js

```diff
@@ -146,6 +146,30 @@ export const StartDraftPermitModal = (props) => {
   );
 
   const isAmendment = props.preDraftFormValues?.type_of_application !== "New Permit";
+  const initialValues = {
+    is_exploration: false,
+    permit_amendment_type_code: props.permitType,
+    type_of_application: props.noticeOfWork?.type_of_application,
+    permit_guid: props.noticeOfWork.source_permit_guid || null,
+    disabled: props.noticeOfWork.source_permit_guid,
+  };
+
+  const sourceAmendmentMessage = props.noticeOfWork.has_source_conditions ? (
+    `This is an amendment to an authorization that was written in Core. All available information, files and conditions have been carried forward from the authorization you are amending.`
+  ) : (
+    <>
+      When you start the Draft Permit, it will include the default conditions for the type of permit
+      you are amending. The Preamble will not include the files from the original authorization. You
+      need to:
+      <br />
+      1. Attach the files that made up the original approved work for the authorization
```

Comment:
Is this suited for a numbered list, or did it look weird?

Reply:
It looks normal: 
![Screen Shot 2021-03-26 at 4 36 14 PM](https://user-images.githubusercontent.com/25125247/112702628-656bf080-8e51-11eb-9679-9b1c368c07cf.png)